corehq/ex-submodules/pillow_retry/tasks.py

```diff
@@ -73,7 +73,7 @@ def process_pillow_retry(error_doc_id):
                     raise Exception('this is temporarily not supported!')
             except ImportError:
                 pass
-            pillow.process_change(change, is_retry_attempt=True)
+            pillow.process_change(change, is_retry_attempt=False)
```

Comment:
@snopoke should this have been `True`? as far as I can tell `is_retry_attempt` is always `False` everywhere it's called (but this one seems like maybe it should be `True`